python/tskit/drawing.py

```diff
@@ -383,6 +383,28 @@ def __init__(
                 class_="x-tick-label",
                 text_anchor="middle",
             )
+        if x_scale == "physical":
+            # Add sites as upper chevrons
+            for s in ts.sites():
+                x = axes_left + s.position * drawing_scale
+                muts = [None] if len(s.mutations) == 0 else s.mutations
+                site = axis.add(dwg.g(class_=f"site s{s.id}"))
+                for i, m in enumerate(muts):
+                    mut = dwg.g(
+                        class_="mut nonvariable" if m is None else f"mut m{m.id}"
```

Comment:
I think we're going through quite a lot of logical contortions here because we're conflating sites and mutations.

Reply:
Are we? I thought this was quite logical. Each position has a site. The site contains (possibly multiple) mutations. The mutations contain a symbol.

It's true that an "empty" site still has a group labelled `<g class="mut nonvariable">...`, which I agree is wrong. We should probably just have a plain symbol without the "mut" class. Perhaps that's what you mean?